source/collections/Queue.ooc

```diff
@@ -77,6 +88,12 @@ VectorQueue: class <T> extends Queue<T> {
 		success := this peek(result&)
 		(result, success)
 	}
+	peek: func ~default (fallback: T) -> T {
+		if (this empty)
+			fallback
+		else
+			this _backend[this _head]
+	}
```

Comment:
This has multiple returns, which we normally don't accept.
```ooc
	peek: func ~default (fallback: T) -> T {
		result: T
		if (this empty)
		 	result = fallback
		else
			result = this _backend[this _head]
		result
	}
```
solves this, but it's longer...